### ISSUE TITLE
Does the ability to apply animations immediately exist?

### ISSUE THREAD

**Issue by ymkim50** [CosmicMind/Motion#8]
I expected the transform to be applied when I applied .duration (0.0) in the motion (...) function, but it did not.

The animation application I want is as follows.

Cell.transform = CGAffineTransform (scaleX: 0, y: 0)
Cell.motion ([. Scale (1), .duration (defaultTimeInterval), .timingFunction (.easeOut)])

I applied the first line to cell.motion (.scale (0), .duration (0)), but internally it seems that the default of duration applies.

If .duration (0), can the transform be applied immediately?

I always appreciate quick answers.

**Comment by daniel-jonathan** [comment]
When you set the value of `0` as the duration, I believe it tells CAAnimation to use the default value. So the idea would be to set the duration to a really small value, such as `0.01`. That will be so fast that it will effectively be immediate. For example:

```swift
fabButton.motion(.scale(2), .duration(0.01), .timingFunction(.easeOut))
```

All the best :)

**Comment by ymkim50** [comment]
I confirmed that it works by the method I have given.

However, this approach can be confusing, so please consider supporting .duration (0).

Thank you for your reply.

**Comment by daniel-jonathan** [comment]
I am going to add this as a feature request :) I agree with you.

**Comment by ymkim50** [comment]
Thank you daniel!

**Comment by daniel-jonathan** [comment]
This is now supported in the next release. Thank you!

**Comment by daniel-jonathan** [comment]
The fix for this issue is in [Motion 1.1.0](https://github.com/CosmicMind/Motion/releases/tag/1.1.0).
